📄 Renamed File for Better Understanding

kadirnar opened the pull request:
@stevenliu and @yiyixu

Renamed the 'rl' file to 'run_locomotion'. This change was made to improve the clarity and readability of the codebase. The 'rl' name was ambiguous, and 'run_locomotion' provides a more clear description of the file's purpose.

Thanks 🙌

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_4056). All of your documentation changes will be reflected on that endpoint.

stevhliu reviewed (commented):
I think the `rl` folder is supposed to be more general for reinforcement learning examples and "locomotion" is just an example of it. cc @patrickvonplaten who may have more insight

pcuenca commented:
I think @stevhliu is right, `rl` is meant to be "reinforcement learning" :)

patrickvonplaten commented on examples/reinforcement_learning/README.md:
```diff
@@ -1,4 +1,4 @@
-# Overview
+"# Overview
```
```suggestion
# Overview
```

stevhliu reviewed (commented):
Thanks! I think the last step is to update the link [here](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) since it'll break when the folder is renamed to `reinforcement_learning`.

patrickvonplaten reviewed (commented):
@stevhliu wdyt?

stevhliu commented:
Ok for me as long as this [link](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) is also updated so nothing breaks :)

kadirnar commented:
> Ok for me as long as this [link](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) is also updated so nothing breaks :)

Sorry. I fixed it. Can you check?
